openfga/v1/authzmodel.proto

```diff
@@ -7,58 +7,61 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 import "validate/validate.proto";
 
 message AuthorizationModel {
-  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
-    example: "{\"id\": \"01G5JAVJ41T49E9TT3SKVS7X1J\", \"type_definitions\":[{\"type\":\"document\",\"relations\":{\"reader\":{\"union\":{\"child\":[{\"this\":{}},{\"computedUserset\":{\"object\":\"\",\"relation\":\"writer\"}}]}},\"writer\":{\"this\":{}}}}]}"
-  };
   string id = 1 [
-    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      example: "\"01G5JAVJ41T49E9TT3SKVS7X1J\""
-    },
-    (validate.rules).string = {
-      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$"
-    }
+    (validate.rules).string = { pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$" },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = { example: '"01G5JAVJ41T49E9TT3SKVS7X1J"' }
   ];
-  repeated TypeDefinition type_definitions = 2 [json_name = "type_definitions"];
-}
 
-message TypeDefinitions {
-  repeated TypeDefinition type_definitions = 1 [
+  repeated TypeDefinition type_definitions = 2 [
     json_name = "type_definitions",
```

Comment:
Why were min items and minimum removed?

Reply:
The whole message TypeDefinitions was deleted and the diff is weird. I did remove the min from `relations`.